docs/src/components/DelayedRenderer.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react'
+import { Loader, Segment, LoaderProps } from '@stardust-ui/react'
+
+type DelayedRendererProps = Partial<{
+  children: JSX.Element
+  placeholder: JSX.Element | LoaderProps['size']
+  visible: boolean
+}>
+
+const paddings: { [key in LoaderProps['size']]: number } = {
+  smallest: 0,
+  smaller: 5,
+  small: 15,
+  medium: 25,
+  large: 40,
+  larger: 65,
+  largest: 100,
+}
+
+const createPlaceholder = (placeholder: DelayedRendererProps['placeholder']): JSX.Element => {
+  const size = typeof placeholder === 'string' ? placeholder : 'medium'
```

Comment:
nit: I would move this bellow line 23 to do it only when necessary (no performance gain, just more understandable code)

Reply:
simplified interface:
- `placeholder` cannot be ReactElement anymore;
- `placeholder` was renamed to `size`